extension/src/experiments/columns/collect/metricsAndParams.ts

```diff
@@ -89,17 +96,28 @@ const walkMetricsOrParamsFile = (
   }
 }
 
+const collectMetricsOrParams = (
+  acc: ColumnAccumulator,
+  type: ColumnType,
+  metricsOrParams: MetricsOrParams | null
+): void => {
+  if (!metricsOrParams) {
+    return
+  }
+  walkMetricsOrParamsFile(acc, type, metricsOrParams)
+}
+
 export const collectMetricsAndParams = (
   acc: ColumnAccumulator,
   data: ExpData
 ) => {
   const { metrics, params } = data
-  if (metrics) {
-    walkMetricsOrParamsFile(acc, ColumnType.METRICS, metrics)
-  }
-  if (params) {
-    walkMetricsOrParamsFile(acc, ColumnType.PARAMS, params)
-  }
+  return Promise.all([
+    // eslint-disable-next-line sonarjs/no-use-of-empty-return-value
```

Comment:
[F] These are false +ves. May need to turn the rule off as we move towards more promises like this.